tests/client/TestPackageJson.js

```diff
@@ -0,0 +1,9 @@
+import packageJson from 'package';
```

Comment:
@kumar303 the rootDir is currently added to jest module paths. I think things might break if I remove it, but I'll see what happens.